src/core/custom-elements/rs-changelog.js

```diff
@@ -82,7 +82,13 @@ async function toHTML(commits) {
     const [message, prNumber = null] = commit.message.split(/\(#(\d+)\)/, 2);
     const commitURL = `${repoURL}commit/${commit.hash}`;
     const prURL = prNumber ? `${repoURL}pull/${prNumber}` : null;
-    const pr = prNumber && hyperHTML` (<a href="${prURL}">#${prNumber}</a>)`;
-    return hyperHTML`<li><a href="${commitURL}">${message.trim()}</a>${pr}</li>`;
+    const pr =
+      prNumber &&
+      html`
+        (<a href="${prURL}">#${prNumber}</a>)
```

Comment:
This is particularly breaking, as hyperHTML does not trim the newlines when the string starts with a plaintext. Maybe worth filing a bug? 🤔

Reply:
There is an existing issue: https://github.com/prettier/prettier/issues/7096